podman/domain/manager.py

```diff
@@ -94,17 +93,14 @@ def exists(self, key: str) -> bool:
         Note:
             This method does _not_ provide any mutex mechanism.
         """
-        raise NotImplementedError()
 
     @abstractmethod
     def get(self, key: str) -> PodmanResourceType:
         """Returns representation of resource."""
-        raise NotImplementedError()
```

Comment:
It is a legal way to provide a "default" implementation. You would call  super().get(...). Since I never found a need, I let the decorator now raise the error.